test/e2e/velero_utils.go

```diff
@@ -302,10 +290,15 @@ func veleroInstall(ctx context.Context, veleroImage string, veleroNamespace stri
 	}
 	err := ensureClusterExists(ctx)
 	if err != nil {
-		return errors.WithMessage(err, "Failed to ensure kubernetes cluster exists")
+		return errors.WithMessage(err, "Failed to ensure Kubernetes cluster exists")
 	}
+
+	providerPlugins := getProviderPlugins(objectStoreProvider)
```

Comment:
This should still happen before the check on line 285. My understanding is that for vSphere we need to install both the vSphere and AWS plugins but at install time you list AWS as the object store provider. It's confusing :)

Reply:
I am glad you caught this, very easy to miss because of naming. I added a short note not to move this call. :)